src/VestRewardsDistributionJob.sol

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Copyright (C) 2021 Dai Foundation
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity 0.8.13;
+
+import {IJob} from "./interfaces/IJob.sol";
+import "./utils/EnumerableSet.sol";
+
+
+interface SequencerLike {
+    function isMaster(bytes32 network) external view returns (bool);
+}
+
+interface DssVestWithGemLike {
+    function unpaid(uint256 _id) external view returns (uint256);
+}
+
+interface VestedRewardsDistributionLike {
+    function distribute() external returns (uint256 amount);
+    function dssVest() external view returns (DssVestWithGemLike);
+    function lastDistributedAt() external view returns (uint256);
+    function vestId() external view returns (uint256);
+}
+
+/// @title Call distribute() when possible
+contract VestRewardsDistributionJob is IJob {
+
+    using EnumerableSet for EnumerableSet.AddressSet;
+
+    // --- Auth ---
+    mapping (address => uint256) public wards;
+
+    function rely(address usr) external auth {
+        wards[usr] = 1;
+
+        emit Rely(usr);
+    }
+    function deny(address usr) external auth {
+        wards[usr] = 0;
+
+        emit Deny(usr);
+    }
+    modifier auth {
+        require(wards[msg.sender] == 1, "VestRewards/not-authorized");
+        _;
+    }
+
+    uint256 public immutable minimumInterval;
```

Comment:
Honestly, I don't believe this is necessary.
The value here will be pretty much arbitrary anyway.

Reply:
removed!